### PR TITLE
[QA-2982][Web][Pipeline & Report] Migrate report files to QAMetrics

### PR DESCRIPTION
**Pull request by stang-wish**
## Background
Jira link: https://jira.wish.site/browse/QA-2982
This task aims to move all QAMetrics Script and related lib files to the new Repo. Meanwhile, the report stage pipeline should be set up.

Related Wiki page:
QAMetrics: https://wiki.wish.site/pages/viewpage.action?spaceKey=QA&title=QA+Automation+Metrics

## Summary
The goal of this PR is temporarily moving all necessary files that I think are needed for now. It also reduced the requirements from QAWebAutomation, and rebuild the base docker image for the pipeline.

Most of the files are copy and paste from QAWebAutomation, some configurations files are different:
.gitigore
.gitlab-ci.yml
CODEOWNERS
requirements.txt
build/base.Dockerfile

No updated wiki pages in this PR.

This change will not affect others work, but for this jira ticket, finally all migrated files should be removed from their original place.

## Validation
No validation for this PR

### CODE REVIEW

File: lib/constants/colors.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class BannerColors:
+    VERIFIED_BY_WISH = "#2FB7EC"
+    WISH_EXPRESS = "#ef8d2e"
+    WISH_EXPRESS_VARIATION = "#EA9C51"
+    LOCAL = "#2eaa77"
```
Comment (judyliu-wish) [inline lib/constants/colors.py:5]: BannerColors and ElementColors are not needed.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+-i https://pypi.infra.wish.com/simple/
+boto3
+google-api-python-client
+google-auth-httplib2
+google-auth-oauthlib
+jinja2
+pre-commit
+prometheus-client
+pylint
+pytest
```
Comment (judyliu-wish) [inline requirements.txt:10]: do you need pytest?

File: lib/constants/colors.py (hunk 1 above)
Comment (judyliu-wish) [reply lib/constants/colors.py:5]: Actually, after doing a quick cmd+f, I don't think this file is needed at all.

File: lib/handlers/google/google_drive_handler.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
+import pickle
```
Comment (judyliu-wish) [inline lib/handlers/google/google_drive_handler.py:1]: This file is broken as the pytest variables are not defined; should be removed instead.

**Review by judyliu-wish — CHANGES_REQUESTED** [review]

File: lib/constants/colors.py (hunk 1 above)
Comment (stang-wish) [reply lib/constants/colors.py:5]: Right, After removed the file_handler.py, this file is not needed.

File: requirements.txt (hunk 2 above)
Comment (stang-wish) [reply requirements.txt:10]: google_drive_handler.py needs

**Review by judyliu-wish — APPROVED** [review]

**Pull request merged**